tests/rbd/test_rbd_group_commands_validation.py

```diff
@@ -0,0 +1,329 @@
+            if err:
+                log.error(f"Create group {group} failed with error {err}")
+                return 1
+            else:
+                log.info(f"Successfully created pool {pool}/{group}")
+
+            # List group
+            out, err = rbd.group.list(**{"pool": pool})
+            if err:
+                log.error(f"Group list failed with error {err}")
+                return 1
+            else:
+                if group in out:
+                    log.info(f"Group {group} listed in pool {pool}")
+                else:
+                    log.error(f"Group {group} not listed in pool {pool}")
+                    return 1
+
+            # Create an RBD image in pool1
+            image = "image_" + random_string(len=4)
+            out, err = rbd.create(**{"image-spec": f"{pool}/{image}", "size": 1024})
+            if err:
+                log.error(f"Create image {pool}/{image} failed with error {err}")
+                return 1
+            else:
+                log.info(f"Successfully created image {pool}/{image}")
+
+            # Add image to the group
+            out, err = rbd.group.image.add(
+                **{"group-spec": f"{pool}/{group}", "image-spec": f"{pool}/{image}"}
+            )
+            if err:
+                log.error(f"Failed to create image {pool}/{image} with error {err}")
+                return 1
+            else:
+                log.info(f"Successfully created image {pool}/{image}")
+
+            # List images in the group
+            out, err = rbd.group.image.list(**{"group-spec": f"{pool}/{group}"})
+            if err:
+                log.error(f"List images in group failed with error {err}")
+                return 1
+            else:
+                if f"{pool}/{image}" in out:
+                    log.info(f"{pool}/{image} listed in group {pool}/{group}")
+                else:
+                    log.error(f"{pool}/{image} not listed in the group image list")
+                    return 1
+
+            # Remove the image  from the group
+            out, err = rbd.group.image.rm(
+                **{"group-spec": f"{pool}/{group}", "image-spec": f"{pool}/{image}"}
+            )
+            if err:
+                log.error(
+                    f"Remove image {pool}/{image} from group {pool}/{group} failed with error {err}"
+                )
+                return 1
+            else:
+                log.info(
+                    f"Successfully removed image {pool}/{image} from group {pool}/{group}"
+                )
+
+            # List images in the group
+            out, err = rbd.group.image.list(**{"group-spec": f"{pool}/{group}"})
+
+            if err:
+                log.error(f"List images in group failed with error {err}")
+                return 1
+            else:
+                if f"{pool}/{image}" not in out:
+                    log.info(f"{pool}/{image} not listed in group {pool}/{group}")
```

Comment:
group spec is used redundant can it be 
`{pool}/{image} not listed in group after successful image deletion`

Reply:
Addressed